1-js/04-object-basics/01-object/article.md

```diff
@@ -101,7 +101,7 @@ For multiword properties, the dot access doesn't work:
 user.likes birds = true
 ```
 
-JavaScript doesn't understand that. It thinks that we address `user.likes`, and then gives a syntax error when comes across unexpected `birds`.
+JavaScript doesn't understand that. It thinks that we're referring to `user.likes`, and then gives a syntax error when it comes across the unexpected `birds`.
```

Comment:
How about 'accessing' (used in [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Property_accessors)):

>JavaScript doesn't understand that. It thinks that we're accessing `user.likes`, and then gives a syntax error when it comes across the unexpected `birds`.